src/jit/compiler.h

```diff
@@ -598,6 +598,20 @@ class  LclVarDsc
         return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
     }
 
+    bool                lvIsMultiregStruct()
+    {
+#if FEATURE_MULTIREG_ARGS_OR_RET
+#ifdef _TARGET_ARM64_
+        if ((TypeGet() == TYP_STRUCT) &&
+            (lvSize()  == 2 * TARGET_POINTER_SIZE))
+        {
+            return true;
+        }
+#endif  // _TARGET_ARM64_
+#endif  // FEATURE_MULTIREG_ARGS_OR_RET
```

Comment:
Just for my info what currently normalizes the local var being passed to be a multiple of TARGET_POINTER_SIZE?  I was surprised to see the '==' rather a '<=' but I'm probably missing the context.